functions/lib/blockiesSVG.js

```
@@ -69,13 +74,30 @@ function buildOptions(opts) {
     return newOpts;
 }
 
-function renderIdenticon(opts) {
+async function renderIdenticon(opts) {
     opts = buildOptions(opts);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)